client.ts

```diff
@@ -11,7 +11,7 @@ import {
 
 const msgidThreshold = 2 ** 32;
 
-type Session = {
+export type SessionForClient = {
```

Comment:
`SessionForClient` in `client.ts` is a bit redundant. It should not be renamed.

Reply:
There is a name conflict in mod.ts, what should I do?

Reply:
Do we need to export `Session` in `client.ts` as public? I thought it was required for internal unit tests.

Reply:
It is argument type of public class.
The types of public arguments of public methods in public classes should be public.

Reply:
> It is argument type of public class.
The types of public arguments of public methods in public classes should be public.

I'm sorry but it doesn't make sense to me. I couldn't imagine developers who use this library need the `SessionForClient` while we expose a real `Session` that is compatible with `SessionForClient`.

Could you explain the reason why you would like to expose the `SessionForClient` a bit more?

Reply:
got it